### ISSUE TITLE
BuildAnalysis duplicates output

### ISSUE THREAD

**Issue by mrlacey** [mrlacey/Rapid-XAML-Toolkit#421]
Running AnalysisExe.exe can lead to output being duplicated.

e.g.

```ascii
RapidXaml.AnalysisExe.exe "C:\Users\matt\source\repos\App195\App195\App195.csproj"
RapidXaml.AnalysisExe, Version=0.11.2.0, Culture=neutral, PublicKeyToken=null
Analyzing C:\Users\matt\source\repos\App195\App195\App195.csproj
- Analyzing: 'C:\Users\matt\source\repos\App195\App195\Styles\Page.xaml'
- Analyzing: 'C:\Users\matt\source\repos\App195\App195\Styles\_Colors.xaml'
- Analyzing: 'C:\Users\matt\source\repos\App195\App195\Styles\_FontSizes.xaml'
- Analyzing: 'C:\Users\matt\source\repos\App195\App195\Styles\_Thickness.xaml'
- Analyzing: 'C:\Users\matt\source\repos\App195\App195\Styles\TextBlock.xaml'
- Analyzing: 'C:\Users\matt\source\repos\App195\App195\Views\MainPage.xaml'
C:\Users\matt\source\repos\App195\App195\Views\MainPage.xaml(54,10): warning Webview2: Use WebView2 instead of WebView (Webview2)
C:\Users\matt\source\repos\App195\App195\Views\MainPage.xaml(54,10): warning Webview2: Use WebView2 instead of WebView (Webview2)
- Analyzing: 'C:\Users\matt\source\repos\App195\App195\Views\ShellPage.xaml'
- Analyzing: 'C:\Users\matt\source\repos\App195\App195\App.xaml'
```

Need to investigate why this is happening and remove duplicates from output.

**Comment by mrlacey** [comment]
The cause here is multiple copies of the library containing the analyzer being found. 
Rather than revisit the code looking for custom analyzer references, choosing to avoid duplicate output being written.
Duplicate output could be produced for other reasons so treating this as a catch-all way of avoiding suspicious duplication in the output/errorList

**Comment by mrlacey** [comment]
released in 0.11.3
